internal/validators/validate.go

```diff
@@ -0,0 +1,80 @@
+package validators
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"unicode"
+
+	"github.com/go-playground/validator/v10"
+	"github.com/stellar/go/strkey"
+)
+
+func NewValidator() *validator.Validate {
+	validate := validator.New()
+	_ = validate.RegisterValidation("public_key", publicKeyValidation)
```

Comment:
RegisterValidation returns an error, right? You should handle it accordingly here by wrapping it and sending the wrapped version to the caller.

Or at least log the error here.

Reply:
It only returns an error if the `key` is empty or the `function` is `nil`. That's why I'm not handling the error here, I don't think we should worry about it, wdyt?